restcomm/restcomm.dao/src/main/java/org/mobicents/servlet/restcomm/dao/DaoUtils.java

```diff
@@ -95,6 +97,14 @@ public static Integer readInteger(final Object object) {
         }
     }
 
+    public static BigInteger readBigInteger(final Object object) {
+        if (object != null) {
+            return (BigInteger) object;
```

Comment:
Noted @jaimecasero .... Please notice all methods throughout this class follow the same design (so, I stuck with it)